envd/api/v1/__init__.py

```diff
@@ -37,7 +37,7 @@ def base(image: str = "ubuntu:20.04", dev: bool = False):
     """Set up the base env.
 
     Args:
-        image (str): docker image, can be any Debian-based images
+        image (str): docker image, can be any Debian-based image
```

Comment:
https://english.stackexchange.com/questions/112996/any-followed-by-singular-or-plural-countable-nouns

Reply:
I thing singular form is better in the context of this argument, because it accepts exactly 1 (one) image for the `image` arg.